app/models/legal_aid_application.rb

```diff
@@ -10,6 +10,12 @@ class LegalAidApplication < ApplicationRecord
 
   validate :proceeding_type_codes_existence
 
+  state_machine initial: :initiated do
```

Comment:
I'd also suggest moving the state machine logic into a separate object and mixing it into the model as my experience is that it can bloat the model very quickly.

Reply:
I don't see any reason, just now, to pre-optimise and move the state machine to a new file. When the bloat happens, I agree, we can move it across to a separate file.